harmonica/tests/test_euler_deconvolution.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2018 The Harmonica Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# This code is part of the Fatiando a Terra project (https://www.fatiando.org)
+#
+import numpy.testing as npt
+import verde as vd
+
+from .. import (
+    EulerDeconvolution,
+    derivative_easting,
+    derivative_northing,
+    derivative_upward,
+    dipole_magnetic,
+    magnetic_angles_to_vec,
+)
+from .._forward.point import point_gravity
+
+
+def test_euler_with_numeric_derivatives():
+    # Add dipole source
+    dipole_coordinates = (10e3, 15e3, -10e3)
+    dipole_moments = magnetic_angles_to_vec(1.0e14, 0, 0)
+
+    # Add regional field
+    inc, dec = -40, 15
+    fe, fn, fu = magnetic_angles_to_vec(1, inc, dec)
+    region = [-100e3, 100e3, -80e3, 80e3]
+    coordinates = vd.grid_coordinates(region, spacing=500, extra_coords=500)
+    be, bn, bu = dipole_magnetic(
+        coordinates, dipole_coordinates, dipole_moments, field="b"
+    )
+
+    # Add a fixed base level
+    true_base_level = 200  # nT
+    anomaly = (fe * be + fn * bn + fu * bu) + true_base_level
+
+    grid = vd.make_xarray_grid(
+        coordinates, anomaly, data_names="tfa", extra_coords_names="upward"
+    )
+    grid["d_east"] = derivative_easting(grid.tfa)
+    grid["d_north"] = derivative_northing(grid.tfa)
+    grid["d_up"] = derivative_upward(grid.tfa)
+    grid_table = vd.grid_to_table(grid)
+    # Verde drops non-dimension coordinates so we have to add z back.
+    # This is a bug in Verde.
+    grid_table["upward"] = grid.upward.values.ravel()
```

Comment:
```suggestion
```

This was fixed in Verde 1.8.1, which is now the minimum required version of Verde.